Userland/Libraries/LibMarkdown/List.cpp

```diff
@@ -37,21 +37,30 @@ DeprecatedString List::render_to_html(bool) const
     return builder.build();
 }
 
-DeprecatedString List::render_for_terminal(size_t) const
+Vector<DeprecatedString> List::render_lines_for_terminal(size_t view_width) const
 {
-    StringBuilder builder;
+    Vector<DeprecatedString> lines;
 
     int i = 0;
     for (auto& item : m_items) {
+        auto item_lines = item->render_lines_for_terminal(view_width);
+        auto first_line = item_lines.take_first();
+
+        StringBuilder builder;
         builder.append("  "sv);
         if (m_is_ordered)
             builder.appendff("{}.", ++i);
         else
             builder.append('*');
-        builder.append(item->render_for_terminal());
+        builder.append(first_line);
+
+        lines.append(builder.build());
+
+        for (auto& line : item_lines)
+            lines.append(move(line));
```

Comment:
This looks like it fails to create proper indentation for the subsequent lines. You can ask the builder directly after line 54 for the amount of indentation you need, I think.